system/velodyne_monitor/Readme.md

```diff
@@ -0,0 +1,63 @@
+# velodyne_monitor
+
+## Purpose
+
+This node monitors the status of Velodyne LiDARs.
+The result of the status is published as diagnostics.
+
+## Inner-workings / Algorithms
+
+The status of Velodyne LiDAR is get from `http://[ip_address]/cgi/{info, settings, status, diag}.json`.
```

Comment:
```suggestion
The status of Velodyne LiDAR can be retrieved from `http://[ip_address]/cgi/{info, settings, status, diag}.json`.
```
Reminder: `is get` seems to not be correct. Might be replaced as suggested or something like that.